build.gradle

```diff
@@ -50,6 +53,37 @@ tasks.withType(ValidatePlugins).configureEach {
     enableStricterValidation = true
 }
 
+// A local repo we publish our library to for testing in order to workaround limitations
+// in the TestKit plugin classpath.
+def localRepo = file("$buildDir/local-repo")
+publishing {
+    // Since for now we need to set `gradlePlugin { automatedPublishing = false ... }`, we have to create manually
```

Comment:
i think lines 60-71 are not required to get this working (to give you a better idea of what is happening here)